package.json

```diff
@@ -12,13 +12,15 @@
     "yarn": ">=0.25.2"
   },
   "dependencies": {
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.3",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
+    "@babel/cli": "^7.0.0-beta.49",
+    "@babel/core": "^7.0.0-beta.49",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.49",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.49",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0-beta.49",
+    "@babel/polyfill": "^7.0.0-beta.49",
+    "@babel/preset-env": "^7.0.0-beta.49",
+    "babel-core": "^7.0.0-0",
```

Comment:
We need to remove the legacy babel-core no?

Reply:
It's just a stub for packages that depend on `babel-core`, see https://www.npmjs.com/package/babel-core/v/7.0.0-bridge.0.
I'll drop it if you wish, but it probably will reduce potential breakage/incompatibility for the time being and it should be dropped later (at least after babel@7 release).

I could make it `^7.0.0-bridge` instead of `^7.0.0-0`, to be more clear.

Reply:
Ahh, I see!  yes please, let's change to `^7.0.0-bridge` for clarity. Thank you